src/api/ratelimits.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_order: 3
 
 Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each unique combination of caller and endpoint.
 
-The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected. We restrict both how frequently a request is made and how many requests are occurring within same time period.
```

Comment:
```suggestion
We restrict both how frequently a request is made and how many concurrent requests a caller can make.

The former rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.  Meanwhile, the concurrency limit will reject requests if the user has too many requests in progress at the same time.
```

I don't know if this is the best way to phrase it. It might be worth getting Angela involved to get the correct wording.